content/docs/user-guide/dvc-files-and-directories.md

```diff
@@ -138,6 +162,56 @@ the possible following fields:
 
 `dvc.yaml` files also support `# comments`.
 
+### dvc.lock file
+
+The purpose of `dvc.lock` is to lock the exact contents of all files involved in
+the pipeline(s) at the time that `dvc.yaml` was last updated.
+
+Here's an example `dvc.lock` based on the one in [`dvc.yaml`](#dvcyaml-file)
+above:
+
+```yaml
+stages:
+  features:
+    cmd: jupyter nbconvert --execute featurize.ipynb
+    deps:
+      - path: data/clean
+        md5: d8b874c5fa18c32b2d67f73606a1be60
+    params:
+      params.yaml:
+        levels.no: 5
+    outs:
+      - path: features
+        md5: 2119f7661d49546288b73b5730d76485
+    metrics:
+      - performance.json
```

Comment:
No metrics or plots in dvc.lock.

Reply:
Oops, thanks. Fixed in dcd09105! (Moved to `outs`.)